metrics.yaml

```diff
@@ -105,7 +105,7 @@ parameters:
 -
   description: 'Specify version for metrics components; e.g. for "openshift/origin-metrics-deployer:latest", set version "latest"'
   name: IMAGE_VERSION
-  value: "3.4.0"
+  value: "v3.5"
```

Comment:
I am going to keep this as v3.5, I think we might want to update our other OCP images to use this naming format as well.